### PR TITLE
For GAWB-1674: Remove all special logic for dbGapAuthorizedUsers authorization domain

### PR DESCRIPTION
**Pull request by MatthewBemis**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [ ] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -15,7 +15,7 @@ object Dependencies {
     "com.getsentry.raven" % "raven-logback" % "7.8.6",
     "com.typesafe.scala-logging" %% "scala-logging-slf4j" % "2.1.2",
     "org.broadinstitute.dsde.vault" %%  "vault-common"  % "0.1-19-ca8b927",
-    "org.broadinstitute.dsde" %%  "rawls-model"  % "0.1-518a712-SNAP"
+    "org.broadinstitute.dsde" %%  "rawls-model"  % "0.1-9847bee-SNAP"
```
Comment (MatthewBemis) [inline project/Dependencies.scala]: note to self: update this if the rawls model changes at all during PR process

**Review by dvoet — APPROVED** [review]
YAY!

File: src/main/scala/org/broadinstitute/dsde/firecloud/model/Workspace.scala [hunk 2]
```diff
@@ -8,71 +8,31 @@ import org.broadinstitute.dsde.rawls.model._
...
-}
-
-/** A Firecloud UI focused result object that performs extra translation on the result from Rawls, specifically
-  * interpreting the NIH realm as a binary "protected" flag.
-  *
-  * Note: Depending on the direction that firecloud-orchestration takes in the future, we may keep this here or move
-  * this logic into firecloud-ui as part of https://broadinstitute.atlassian.net/browse/GAWB-1674. See discussion in
-  * https://github.com/broadinstitute/firecloud-orchestration/pull/388.
-  */
-case class UIWorkspace(
```
Comment (MatthewBemis) [inline src/main/scala/org/broadinstitute/dsde/firecloud/model/Workspace.scala:58]: @helgridly i was told that you'd be very happy to see this finally die!

Comment (helgridly) [reply src/main/scala/org/broadinstitute/dsde/firecloud/model/Workspace.scala:58]: I am! Thank you for letting me know!

File: project/Dependencies.scala (hunk 1 above)
Comment (helgridly) [reply project/Dependencies.scala]: you should manually publish a new rawlsModel anyway; pointing to -SNAP libraries should only be done during local development (though in practise it doesn't really matter at all)

**Pull request merged**